logging/import-logs/main.py

```diff
@@ -79,6 +79,18 @@ def _day(blob_name: str) -> int:
     return int(blob_name[offset : offset + 2])
 
 
+def _is_valid_import_range() -> bool:
```

Comment:
nit: for consistency with other functions in this file please add a docstring explaining what this function is for.

Reply:
I will move the comment to the docstring. Can you confirm that private methods with few lines of code still require a docstring?

Reply:
I don't see a strict rule in sample guide, but taking into account that all other functions had such docstring (even private ones) I have noticed it.